meta/documents/user_guide/de/obtaining_credentials.md

```diff
@@ -0,0 +1,48 @@
+# Wayfair-Plugin: API-Anmeldeinformationen abrufen
+
+Damit das Wayfair-Plugin eine Verbindung zu Wayfair-Systemen herstellen kann, müssen Sie Ihre eindeutigen API-Anmeldeinformationen angeben.
+
+Gehen Sie folgendermaßen vor, um die API-Anmeldeinformationen für Ihre Organisation zu erhalten:
+
+## 1. Senden Sie eine E-Mail an Wayfair
+
+Bitte senden Sie eine E-Mail an Ihren Wayfair-Account Manager und kopieren Sie (CC) ERPSupport@wayfair.com, um Unterstützung anzufordern:
+
+- Betreff: Zugriff auf das Plentymarkets-Plugin / (Name Ihres Unternehmens) (Wayfair-Lieferanten-ID)
+- Inhalt:
+    - Ihre Kontaktinformationen
+    - kurze Beschreibung, wie Sie das Plentymarkets-Plugin verwenden möchten
+    - Datum, an dem Sie bereit sind das Wayfair Plentymarkets Plugin zu verwenden
+
+Sie erhalten umgehend eine Antwort mit folgenden Angaben:
+- Bestätigung Ihres eingerichteten API-Zugriffs.
+- Lieferanten-ID(s) für Ihre Lager.
+
+## 2. Anmeldeinformationen für die Anwendung generieren
+
+1. Gehen Sie zu https://partners.wayfair.com/developer/applications
+
+2. Geben Sie Ihre Wayfair Partner Home-Anmeldeinformationen ein. Sie sollten zur Seite "Anwendungsverwaltung" für Wayfair weitergeleitet werden.
+
+3. Klicken Sie auf der Seite "Anwendungsverwaltung" unten auf die Schaltfläche "+ Neue Anwendung". ![new application button](../../../images/de/wayfair_app_management/button_new_app.png)
+
+4. Geben Sie auf dem Formular "Neue Bewerbung" die Details der neuen Bewerbung an:
+    * `Name`: Eine nützliche Kennung wie "[Name des Lieferanten] Plentymarkets Plugin".
+    * `Description`: so etwas wie "Plentymarkets Plugin-Anwendung für [Name des Lieferanten] Go-Live im Herbst 2020".
+
+5. Verwenden Sie den Schieberegler am unteren Rand des Dialogfelds, um ihn auf "Produktion" zu setzen, sofern Wayfair nichts anderes anordnet.
+
+6. Klicken Sie im Dialogfeld auf "Speichern", um die Anmeldeinformationen der Anwendung anzuzeigen - `Kunden-ID` und `Kundengeheimnis`.
+

```

Comment:
Check if there are things in "quotes" here that are supposed to be in `back-ticks`, especially in steps 2.2 and 2.3.